pyproject.toml

```diff
@@ -18,9 +18,10 @@ classifiers = [
 include = ["sphinxcontrib/autodoc_pydantic/css/autodoc_pydantic.css"]
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<4.0.0"
-Sphinx = ">=3.4"
-pydantic = ">=1.5,<2.0.0"
+python = ">=3.8.0,<4.0.0"
```

Comment:
Was there a specific reason to remove support for python 3.7? It is EOL by June 2023, however I would like to keep backwards compatibility if possible.

Reply:
I thought pydantic would remove py37 support after seeing https://github.com/pydantic/pydantic/issues/6426 but they fixed the issue. I'll add py37 back, there is a workaround needed around `importlib_metadata` -> `importlib.metadata`